tests/test_translator.py

```diff
@@ -277,6 +281,105 @@ def test_parse_raw_request_no_path(self):
         with self.assertRaises(ValueError):
             translator.parse_raw_request(raw_request)
 
+    def test_process_arguments_with_no_arguments(self):
+        args = load_args()
+
+        with self.assertRaises(SystemExit) as exc:
+            translator.process_arguments(args)
+
+        self.assertEqual(exc.exception.code, -1)
+
+    def test_process_arguments_with_interactive_mode(self):
+        args = load_args(language=["bash"], interactive=True)
+
+        with self.assertRaises(SystemExit) as exc:
+            with mock.patch('http_request_translator.translator.take_headers', side_effect=KeyboardInterrupt):
+                translator.process_arguments(args)
+
+        self.assertEqual(exc.exception.code, 0)
+
+    def test_process_arguments_with_request(self):
+        request_data = """GET  HTTP/1.1\nHost: google.com\nCache-Control: no-cache"""
+        args = load_args(language=["bash"], request=request_data)
+
+        self.assertEqual(type(translator.process_arguments(args)), type({}))
+
+    def test_process_arguments_with_file(self):
+        request_data = """GET  HTTP/1.1\nHost: google.com\nCache-Control: no-cache"""
+        with open('temp', 'w+') as f:
+            f.write(request_data)
+
+        args = load_args(language=["bash"], file='temp')
+
+        self.assertEqual(type(translator.process_arguments(args)), type({}))
+        os.remove('temp')
+
+    def test_process_arguments_with_no_language(self):
+        request_data = """GET  HTTP/1.1\nHost: google.com\nCache-Control: no-cache"""
+        args = load_args(request=request_data)
+
+        self.assertEqual(type(translator.process_arguments(args)), type({}))
+
+    def test_process_arguments_with_multiple_language(self):
+        request_data = """GET  HTTP/1.1\nHost: google.com\nCache-Control: no-cache"""
+        args = load_args(language=["bash", "php", "python"], request=request_data)
+
+        self.assertEqual(type(translator.process_arguments(args)), type({}))
+
+    def test_process_arguments_with_wrong_language(self):
+        request_data = """GET  HTTP/1.1\nHost: google.com\nCache-Control: no-cache"""
+        args = load_args(language=["lua"], request=request_data)
```

Comment:
Who knows, maybe one day we will support LUA. Could you use a random-ish value instead?